src/YourEasyRent/UserState/UserSearchState.cs

```diff
@@ -19,24 +19,36 @@ public IEnumerable<MenuStatus> HistoryOfMenuStatuses
 
         private List<MenuStatus> _historyOfMenuStatuses = new List<MenuStatus>();
```

Comment:
and created IEnumerable<MenuStatus> HistoryOfMenuStatuses like property in USS class